### ISSUE TITLE
addAsyncValidator: Utils is not defined

### ISSUE THREAD

**Issue by cyrilfr** [guillaumepotier/Parsley.js#1198]
I have an error when I try to add an async validator like this :

```
$("#myform").parsley().on("form:submit", function(form)
{
    return form.isValid();
}
).addAsyncValidator("login-validator", function(xhr)
{
    if (xhr.status == "200") return true;
},
"/user/login-validator");
```
Here is the error I get:

```
jQuery.Deferred exception: Utils is not defined window.ParsleyExtend.addAsyncValidator@http://localhost:9000/assets/javascripts/plugins/parsley/parsley.min.js:17:7512
initParsley@http://localhost:9000/assets/javascripts/entities/user.js:51:3
init@http://localhost:9000/assets/javascripts/entities/user.js:21:3
@http://localhost:9000/user/profile:1123:3
j@http://localhost:9000/assets/javascripts/jquery-3.1.1.min.js:2:29946
g/</k<@http://localhost:9000/assets/javascripts/jquery-3.1.1.min.js:2:30262
 undefined

ReferenceError: Utils is not defined
```

**Comment by marcandre** [comment]
Thanks for the report, and sorry about this.

You were meant to get a console message stating:

> Accessing the method `addAsyncValidator` through an instance is deprecated. Simply call `Parsley.addAsyncValidator(...)

If you do so, you will not get an error. I will fix the code in the meantime.

**Comment by marcandre** [comment]
Fixed in 2.7.1
